core/operation.ts

```diff
@@ -193,6 +193,7 @@ export class Operation {
 
     const appliedQueries = context.apply(this.contextableQueries);
     this.proto.queries = typeof appliedQueries === "string" ? [appliedQueries] : appliedQueries;
+    this.proto.queries.forEach(query => validateQueryString(this.session, query));
```

Comment:
I added this for operations too, but it's not actually needed is it, because we never append SQL to the end of operations? WDYT?

Reply:
yeah personally, I liked the way you had it originally - leave operations as-is, because we never futz with them.